PyInstaller/__init__.py

```diff
@@ -26,24 +25,6 @@
 
 HOMEPATH = os.path.dirname(PACKAGEPATH)
 
-# Update __version__ as necessary.
-if os.path.exists(os.path.join(HOMEPATH, 'setup.py')):
-    # PyInstaller is run directly from source without installation, or __version__ is called from 'setup.py'...
-    if compat.getenv('PYINSTALLER_DO_RELEASE') == '1':
-        # Suppress the git revision when doing a release.
-        pass
-    elif 'sdist' not in sys.argv:
-        # and 'setup.py' was not called with 'sdist' argument. For creating source tarball we do not want git revision
-        # in the filename.
-        try:
-            __version__ += get_repo_revision()
```

Comment:
I think this was the only use of `PyInstaller.utils._gitrevision` (which supplies git commit/revision information for archive snapshots generated via git-archive), so we can remove that as well. (Plus the corresponding [entry in .gitattributes](https://github.com/pyinstaller/pyinstaller/blob/5bd9dd70b4c792103d048ecf43933ba8d71e0993/.gitattributes#L25))